psmdb-tarball/psmdb-tarball/tests/test_psmdb_tarball.py

```diff
@@ -32,8 +31,20 @@ def test_jstests(host, jstest):
 
 @pytest.mark.parametrize("suites", SUITES)
 def test_suites(host, suites):
+    if host.system_info.distribution == "debian" or (host.system_info.distribution == "ubuntu" and "24.04" in host.system_info.release):
+        pytest.skip("Skip debian12 as no openssl with FIPS available")
     cmd = "cd /percona-server-mongodb && /opt/venv/bin/python buildscripts/resmoke.py run --suite "  + suites
     with host.sudo():
         result = host.run(cmd)
         print(result.stderr)
     assert result.rc == 0, result.stdout
+
+@pytest.mark.parametrize("fips", FIPS)
```

Comment:
We can test FIPS on regular Al2023/RHEL 8/9 but it requires ubuntu jammy/noble PRO to use FIPS.